test/cpp_extensions/open_registration_extension/pytorch_openreg/__init__.py

```diff
@@ -11,6 +11,9 @@
 import pytorch_openreg._C  # noqa: F401  # usort: skip
 
 
+""" TODO Add back and rename all 'foo' to 'openreg'
```

Comment:
This might be the cause of the CI failures?
This for sure break test_openreg.py

Reply:
There is already `_register_device_module` in ut with different implementation, so I commented it out here. I have rolled back the code here and monkey patch _OpenRegMod in ut.